src/test/java/GreetzLoginFavouritesTest.java

```diff
@@ -43,103 +55,79 @@ public boolean isLoggedIn(String firstName) {
     }
 
-    @Test
+    @Test(priority = 5)
     public void testCardsPricingChange() throws InterruptedException {
-        driver = createDriverInstance();
-        try {
-            testLoginImpl();
 
-            driver.get("https://www.greetz.nl/kaarten/denken-aan");
-            Thread.sleep(3000);
-
-            List<WebElement> cards = driver.findElements(By.xpath("//div[@class=\"b-card-preview__container\"]"));
-            WebElement randomChosenItem = randomItem(cards);
-            randomChosenItem.click();
-            Thread.sleep(3000);
+        driver.get("https://www.greetz.nl/kaarten/denken-aan");
+        Thread.sleep(3000);
 
-            WebElement randomChosenItemQuantity = driver.findElement(By.xpath("//input[@type='number']"));
-            int chosenQuantity = getRandomNumberUsingNextInt(10) + 1;
-            randomChosenItemQuantity.clear();
-            randomChosenItemQuantity.sendKeys("" + chosenQuantity);
-            Thread.sleep(2000);
+        List<WebElement> cards = driver.findElements(By.xpath("//div[@class=\"b-card-preview__container\"]"));
+        WebElement randomChosenItem = randomItem(cards);
+        randomChosenItem.click();
+        Thread.sleep(3000);
 
-            String price = driver.findElement(By.xpath("//*[@class='price-normal']")).getText();
-            double priceOfItem = parsePriceStrToDouble(price);
+        WebElement randomChosenItemQuantity = driver.findElement(By.xpath("//input[@type='number']"));
+        int chosenQuantity = getRandomNumberUsingNextInt(10) + 1;
+        randomChosenItemQuantity.clear();
+        randomChosenItemQuantity.sendKeys("" + chosenQuantity);
+        Thread.sleep(2000);
 
-            double expectedTotalPrice = chosenQuantity * priceOfItem;
+        String price = driver.findElement(By.xpath("//*[@class='price-normal']")).getText();
+        double priceOfItem = parsePriceStrToDouble(price);
 
-            String actualTotalPriceLine = driver.findElement(By.xpath("//div[@class='price-total']")).getText();
-            double actualTotalPrice = Double.parseDouble(actualTotalPriceLine.split(" ")[2].replace(',', '.'));
-//            .substring(7, 7 + expectedTotalPriceFinal.length());
+        double expectedTotalPrice = chosenQuantity * priceOfItem;
 
-            Assert.assertEquals(actualTotalPrice, expectedTotalPrice, 0.001, "Total prices: ");
-            Thread.sleep(2000);
+        String actualTotalPriceLine = driver.findElement(By.xpath("//div[@class='price-total']")).getText();
+        double actualTotalPrice = Double.parseDouble(actualTotalPriceLine.split(" ")[2].replace(',', '.'));
 
-        } finally {
-            driver.quit();
-        }
+        Assert.assertEquals(actualTotalPrice, expectedTotalPrice, 0.001, "Total prices: ");
+        Thread.sleep(2000);
     }
 
     //not original
-    private int getRandomNumberUsingNextInt(double max) {
+    private int getRandomNumberUsingNextInt(int max) {
```

Comment:
```suggestion
    private int getRandomNumber(int max)
```

Reply:
Addressed in 5811ad0.

See [131](https://github.com/Shake-Ghairbyan/epam_java_automation/pull/6/files#diff-5903713c83ae56134371728d87051c0d93bf2876c6918c072e9709c7ef3d0453R131)